### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Sensationull**
CSS Modules
@kleekich

### CODE REVIEW

File: client/src/components/EstimatedCost.jsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import form from './Form.css';
 
 const EstimatedCost = (props) => (
-  <div> Estimated Cost
-    <span id="estimated-cost-calc"> {props.estimation}</span>
+  <div className={form.estimatedPriceContainer}> 
+    <div>Estimated Cost
```
Comment (kleekich) [inline client/src/components/EstimatedCost.jsx:6]: You might want to wrap a span tag for 'Estimated Cost' maybe?

File: client/src/components/Form.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+/* .form {
```
Comment (kleekich) [inline client/src/components/Form.css:1]: You might want to remove comment unless you want to save it for later.

File: client/src/components/MarketPrice.jsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-
+import form from './Form.css';
 const MarketPrice = (props) => (
-  <div>
-    <button id="market-price-button">Market Price </button>
-    <span id="market-price-number">${props.prices}</span> 
+  <div className= {form.marketPriceContainer}>
+    <div className= {form.marketPriceTitle}> Market Price
```
Comment (kleekich) [inline client/src/components/MarketPrice.jsx:5]: Maybe you want to wrap a span tag for 'Market Price'?

**Comment by kleekich** [comment]
@andrewjohnso7 Other than only few comments on style, everything looks good!

**Review by kleekich — APPROVED** [review]
@andrewjohnso7 Other than only few comments on style, everything looks good!

File: client/src/components/Form.css (hunk 2 above)
Comment (Sensationull) [reply client/src/components/Form.css:1]: Thanks, I'll be refactoring the CSS appropriately

File: client/src/components/MarketPrice.jsx (hunk 3 above)
Comment (Sensationull) [reply client/src/components/MarketPrice.jsx:5]: I'm not sure I follow?

File: client/src/components/EstimatedCost.jsx (hunk 1 above)
Comment (Sensationull) [reply client/src/components/EstimatedCost.jsx:6]: I'll look into it

**Pull request merged**